com.unity.render-pipelines.high-definition/Documentation~/Alpha-Output.md

```diff
@@ -2,7 +2,8 @@
 
 To maximize performance and minimize bandwidth usage, HDRP by default renders image frames in the **R11G11B10** format. However, this format does not include an alpha channel, which might be required for applications that want to composite HDRP's output over other images.
 
-In order to configure HDRP to output an alpha channel, the user should first choose the  **R16G16B16A16** *Color Buffer Format* in the rendering section of the HDRP Asset, which can be found in the [ Edit -> Project Settings ] menu. However, please note that enabling this option will incur a performance overhead.
+In order to configure HDRP to output an alpha channel, the user should first choose the  **R16G16B16A16** *Color Buffer Format* in the rendering section of the HDRP Asset, which can be found in the [ Edit -> Project Settings ] menu. However, please note that enabling this option will incur a performance overhead. In HDRP, opaque materials always output 1 in the alpha channel, unless [Alpha Clipping](Alpha-Clipping) is enabled.
```

Comment:
To configure HDRP to output an alpha channel, open your [HDRP Asset](HDRP-Asset.md) (menu: **Edit > Project Settings > Graphics > Scriptable Render Pipeline Asset**), go to the **Rendering** section, and set the **Color Buffer Format** to **R16G16B16A16**. However, note that enabling this option incurs a performance overhead. In HDRP, opaque materials always output 1 in the alpha channel, unless you enable [Alpha Clipping](Alpha-Clipping.md).